lib/autoproj/ops/install.rb

```diff
@@ -52,8 +52,16 @@ def initialize(root_dir)
                 @ruby_executable = config['ruby_executable']
                 @local = false
 
+                xdg_default_gem_path = File.join(Dir.home, '.local', 'share', 'autoproj', 'gems')
```

Comment:
Why isn't this XDG_DATA_HOME instead? Same for the bootstrap/install scripts..

Ref: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

Reply:
This is XDG_DATA_HOME.

Unfortunately, the xdg gem actually does not handle the environment variables - it only handles the default XDG folders in Dir.home. Moreover, install.rb should be dependency-free (it ends up included in the install/bootstrap script) and therefore cannot use the xdg gem.

Reply:
> Unfortunately, the xdg gem actually does not handle the environment variables

Well .. this is actually not true.

Reply:
... and I therefore added support for the XDG_DATA_HOME env var.